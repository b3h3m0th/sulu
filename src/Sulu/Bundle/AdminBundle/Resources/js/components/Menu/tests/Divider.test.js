// @flow
import {render} from '@testing-library/react';
import React from 'react';
import Divider from '../Divider';

test('The component should render', () => {
    const {container} = render(<Divider />);
    expect(container).toMatchSnapshot();
});
