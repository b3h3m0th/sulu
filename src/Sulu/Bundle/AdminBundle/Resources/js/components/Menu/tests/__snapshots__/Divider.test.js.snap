// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The component should render 1`] = `
<div>
  <li
    class="divider"
  />
</div>
`;
